src/content/docs/apm/agents/java-agent/additional-installation/install-java-agent-java-2-security.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Java 2 security installation for WebLogic and WebSphere
+tags:
+  - Agents
+  - Java agent
+  - Additional installation
+metaDescription: 'For WebLogic and WebSphere users with Java 2 Security or Administrative Security: Additional install steps for the New Relic Java agent.'
+redirects:
+  - /docs/agents/java-agent/installation/install-java-agent-java-2-security
+  - /docs/apm/agents/java-agent/installation/install-java-agent-java-2-security
+---
+
+If you're a WebLogic or WebSphere user and use the Java 2 security model, our Java agent requires configuration in addition to the [Java agent installation](/install/java).
+
+## WebLogic: Java 2 Security [#weblogic-java-2]
+
+If you're using Java 2 security, add the following entry to `weblogic.policy` to complete installation:
+
+```
+grant codeBase "file:<var>/full/path/to/</var>newrelic/-" {
+    permission java.security.AllPermission;
+  };
+```
+
+Specify the directory containing `newrelic.jar` in `/full/path/to/newrelic`. The default policy file location is `weblogic_root/server/lib/weblogic.policy`.
+
+## WebSphere: Java 2 Security or Administrative Security [#websphere-java-2]
+
+If you are using Java 2 Security or WebSphere Admin Security, you must grant all jars inside the New Relic install folder the proper permissions. 
+
+To enable New Relic for all app servers, edit the `java.policy` file:
+
+1. Open your `java.policy` file, located inside your WebSphere install directory at:
+
+    ```
+    <var>YOUR_WEBSPHERE_INSTALL_DIRECTORY</var>/java/jre/lib/security/java.policy
```

Comment:
@zuluecho9 We're not using these var tags anymore https://docs.newrelic.com/docs/style-guide/formatting/code-examples/